src/Akeneo/Tool/Bundle/MeasureBundle/Controller/ExternalApi/SaveMeasurementFamiliesAction.php

```diff
@@ -0,0 +1,154 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Akeneo\Tool\Bundle\MeasureBundle\Controller\ExternalApi;
+
+use Akeneo\Tool\Bundle\MeasureBundle\Application\SaveMeasurementFamily\SaveMeasurementFamilyCommand;
+use Akeneo\Tool\Bundle\MeasureBundle\Application\SaveMeasurementFamily\SaveMeasurementFamilyHandler;
+use Akeneo\Tool\Bundle\MeasureBundle\Controller\ExternalApi\JsonSchema\MeasurementFamilyListValidator;
+use Akeneo\Tool\Bundle\MeasureBundle\Controller\ExternalApi\JsonSchema\MeasurementFamilyValidator;
+use Akeneo\Tool\Component\Api\Exception\ViolationHttpException;
+use Akeneo\Tool\Component\Api\Normalizer\Exception\ViolationNormalizer;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Exception\BadRequestHttpException;
+use Symfony\Component\Validator\Validator\ValidatorInterface;
+
+/**
+ * @author    Samir Boulil <samir.boulil@akeneo.com>
+ * @copyright 2020 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class SaveMeasurementFamiliesAction
+{
+	/** * @var MeasurementFamilyListValidator */
+	private $measurementFamilyListValidator;
+
+	/** @var MeasurementFamilyValidator */
+	private $measurementFamilyStructureValidator;
+
+	/** @var ValidatorInterface */
+	private $validator;
+
+	/** @var ViolationNormalizer */
+	private $violationNormalizer;
+
+	/** @var SaveMeasurementFamilyHandler */
+	private $saveMeasurementFamilyHandler;
+
+	/** @var int */
+	private $maximumMeasurementFamiliesPerRequest;
+
+	public function __construct(
+		MeasurementFamilyListValidator $measurementFamilyListValidator,
+		MeasurementFamilyValidator $measurementFamilyStructureValidator,
+		ValidatorInterface $validator,
+		ViolationNormalizer $violationNormalizer,
+		SaveMeasurementFamilyHandler $saveMeasurementFamilyHandler,
+		int $maximumMeasurementFamiliesPerRequest
+    ) {
+		$this->measurementFamilyListValidator = $measurementFamilyListValidator;
+		$this->measurementFamilyStructureValidator = $measurementFamilyStructureValidator;
+		$this->validator = $validator;
+		$this->violationNormalizer = $violationNormalizer;
+		$this->saveMeasurementFamilyHandler = $saveMeasurementFamilyHandler;
+		$this->maximumMeasurementFamiliesPerRequest = $maximumMeasurementFamiliesPerRequest;
+	}
+
+	public function __invoke(Request $request): Response
```

Comment:
```suggestion
	public function handle(Request $request): Response
```
Didn't we decide to not use `invoke`?

Reply:
for application handler yes, for controllers it's a different story. first because we rarely call controllers ourselves, it's the job of the framework and we used this to define our controllers https://symfony.com/doc/current/controller/service.html#invokable-controllers
For application services, this is our code, that we will be responsible for coding in this case __invoke is not convinient.
But honestly, we could not use __invoke here and use indexAction or something stated in the docs